<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lbl_query_liveuser">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>本工具提供优卡活跃用户数查询：</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="date_from_last">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="date_end_last">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_date_from_last">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>上周起始日期</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_date_end_last">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>32</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>上周结束日期</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_query">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_export_excel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>导出到Excel</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tb_liveuser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>881</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_date_from">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>起始日期</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="date_end">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>70</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_date_end">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>结束日期</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="date_from">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_make_chart">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>生成EXCEL图表</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
